### ISSUE TITLE
UnsatisfiedLinkError with jniFlyCapture2.dll

### ISSUE THREAD

**Issue by fahruz** [bytedeco/javacpp-presets#403]
This is similar to issue #337, but I get different different DLLs missing from Dependency Walker.

![image](https://cloud.githubusercontent.com/assets/21228304/24385130/4df974a8-1336-11e7-893f-7a45e6aa7f3d.png)

This is the output from the JVM:

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: no jniFlyCapture2 in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:804)
	at org.bytedeco.javacpp.Loader.load(Loader.java:613)
	at org.bytedeco.javacpp.Loader.load(Loader.java:530)
	at org.bytedeco.javacpp.FlyCapture2.<clinit>(FlyCapture2.java:10)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.bytedeco.javacpp.Loader.load(Loader.java:585)
	at org.bytedeco.javacpp.Loader.load(Loader.java:530)
	at org.bytedeco.javacpp.FlyCapture2$FC2Version.<clinit>(FlyCapture2.java:709)
	at FlyCapture2Test.PrintBuildInfo(FlyCapture2Test.java:28)
	at FlyCapture2Test.main(FlyCapture2Test.java:135)
Caused by: java.lang.UnsatisfiedLinkError: C:\Users\joe\.javacpp\cache\flycapture-2.9.3.43-1.3-windows-x86_64.jar\org\bytedeco\javacpp\windows-x86_64\jniFlyCapture2.dll: Can't find dependent libraries
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:784)
	... 10 more
 
```
My OS is Win7 and I have the latest version of the FlyCapture SDK.

**Comment by jpsacha** [comment]
If you have FlyCapture 2.10.+ it may be an issue. Current release of the presets supports 2.9. See also PR #400

**Comment by fahruz** [comment]
Thanks for the reply. That's correct. I have FlyCapture v2.10.3.266.

**Comment by saudet** [comment]
With JavaCPP 1.4, binaries have been released for 2.11, and they should be working well:
http://search.maven.org/#search%7Cga%7C1%7Cbytedeco%20flycapture
If you have any issues with them though, please let me know! 

@jpsacha Thanks for maintaining those presets!
